src/hooks/useInitializeENSIntroData.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback } from 'react';
+import { prefetchENSProfile } from './useENSProfile';
+import { prefetchENSProfileImages } from './useENSProfileImages';
+import { prefetchENSResolveName } from './useENSResolveName';
+import { ensIntroMarqueeNames } from '@rainbow-me/references';
+
+export default function useInitializeENSIntroData() {
+  const initializeENSIntroData = useCallback(async () => {
+    await Promise.all(
+      ensIntroMarqueeNames.map(async name => {
+        prefetchENSResolveName(name);
+        prefetchENSProfileImages({ name });
+        prefetchENSProfile({ name });
```

Comment:
this is more a question but i see `prefetchENSProfile` has `fetchImages` inside, and at the same time `prefetchENSProfileImages` has `fetchImages` inside, so we're asking for the intro profiles images twice on each app load. wondering if we could improve this?

Reply:
hmm not 100% sure but i think since the query has a stale time of 10s it wouldn't actually fetch the images a second time. but if that's true then i think `prefetchENSProfileImages` is redundant